exporters/otlp/otlpmetric/exporter_test.go

```diff
@@ -63,8 +63,8 @@ func (m *stubClient) Stop(ctx context.Context) error {
 	return nil
 }
 
-func (m *stubClient) UploadMetrics(ctx context.Context, protoMetrics []*metricpb.ResourceMetrics) error {
-	m.rm = append(m.rm, protoMetrics...)
+func (m *stubClient) UploadMetrics(ctx context.Context, protoMetrics *metricpb.ResourceMetrics) error {
+	m.rm = append(m.rm, []*metricpb.ResourceMetrics{protoMetrics}...)
```

Comment:
```suggestion
	m.rm = append(m.rm, protoMetrics)
```

It shouldn't be necessary to make a slice just to destructure it here.